panel/src/components/Forms/Input/DateInput.vue

```diff
@@ -25,18 +24,8 @@
 <script>
 import Input, { props as InputProps } from "@/mixins/input.js";
 
-export const props = {
-	mixins: [InputProps],
+export const IsoProps = {
```

Comment:
TBH I stumbled across IsoProps. Is this clear enough? Maybe IsoDateProps instead?